AcdhArcheAssets/formats.json

```diff
@@ -712,87 +1564,105 @@
       "Magic_number" : "",
-      "PRONOM_ID" : "",
+      "PRONOM_ID" : "fmt/353",
       "comment" : "",
+      "Wikidata-ID" : "Q27231633"
       "extensions" : [
          "tif",
          "tiff"
       ],
-      "name" : "Baseline TIFF v. 6, uncompressed"
+      "name" : "Tag(ged) Image File FormatBaseline v. 6, uncompressed"
    },
    {
       "ARCHE_category" : "https://vocabs.acdh.oeaw.ac.at/archecategory/text",
       "ARCHE_docs" : "",
       "ARCHE_conformance": "preferred",
+      "DataType" : "Text",
+      "formatVersion" : "",
+      "LoCFDD": "",
       "Further_specifications" : "",
-      "IANA_reference" : "",
+      "IANA_reference" : "[RFC2046, RFC3676, RFC5147",
       "IANA_template" : "",
-      "Informal_MIME_type" : [],
+      "Informal_MIME_type" : [application/text],
       "Long_term_format" : "",
       "MIME_type" : [
          "text/plain"
       ],
       "Magic_number" : "",
-      "PRONOM_ID" : "",
+      "PRONOM_ID" : "x-fmt/111",
       "comment" : "",
+      "Wikidata-ID" : "Q1145976"
       "extensions" : [
          "txt"
       ],
       "name" : "Plain Text"
    },
    {
       "ARCHE_category" : "https://vocabs.acdh.oeaw.ac.at/archecategory/3dData",
-      "ARCHE_docs" : "",
+      "ARCHE_docs" : "3D & Virtual Reality",
       "ARCHE_conformance": "preferred",
+      "DataType" : "3D Vector Graphics file",
+      "formatVersion" : "",
+      "LoCFDD": "fdd000490",
       "Further_specifications" : "",
       "IANA_reference" : "",
-      "IANA_template" : "",
-      "Informal_MIME_type" : [],
+      "IANA_template" : "Web3D, Web3D_X3D",
+      "Informal_MIME_type" : [model/x3d+fastinfoset],
       "Long_term_format" : "",
-      "MIME_type" : [],
+      "MIME_type" : [model/x3d+xml, model/x3d+binary, model/x3d+vrml],
       "Magic_number" : "",
-      "PRONOM_ID" : "",
+      "PRONOM_ID" : "fmt/579, fmt/580, fmt/581, fmt/582",
       "comment" : "",
+      "Wikidata-ID" : "Q930428"
       "extensions" : [
          "x3d"
       ],
       "name" : "eXtensible 3D Graphics"
    },
    {
       "ARCHE_category" : "https://vocabs.acdh.oeaw.ac.at/archecategory/dataset",
-      "ARCHE_docs" : "",
+      "ARCHE_docs" : "Text Documents",
       "ARCHE_conformance": "preferred",
+      "DataType" : "Text format",
+      "formatVersion" : "",
+      "LoCFDD": "",
       "Further_specifications" : "",
-      "IANA_reference" : "",
-      "IANA_template" : "",
+      "IANA_reference" : "RFC7303",
+      "IANA_template" : "https://www.iana.org/assignments/media-types/text/xml",
       "Informal_MIME_type" : [],
       "Long_term_format" : "",
       "MIME_type" : [
          "text/xml",
          "application/xml"
       ],
       "Magic_number" : "",
-      "PRONOM_ID" : "",
+      "PRONOM_ID" : "fmt/101",
       "comment" : "https://vocabs.acdh.oeaw.ac.at/archecategory/dataset/xml",
+      "Wikidata-ID" : "Q2115"
       "extensions" : [
          "xml"
       ],
-      "name" : "eXtensible Markup Language"
```

Comment:
Der ursprüngliche Wert passt so. Das X ist groß, um zu zeigen wo das x in xml herkommt